config/passport.js

```diff
@@ -5,7 +5,7 @@ var ExtractJwt = require('passport-jwt').ExtractJwt;
 
 // load up the user model
 var User = require('../app/models/user');
-var config = require('../config/config'); // get db config file
+var config = require('../config/config.dev.js'); // get db config file
```

Comment:
So this isn't correct. In our source files we have config.dev.js. Gulp renames this to config.js. The reason for this is we will have multiple config files - for example, we might have config.prod.js. By specifying the environment when running gulp, it will automagically select the right configuration for what we want to use